src/components/flag/code/views/flagView.jsx

```diff
@@ -3,12 +3,12 @@ var React = require('react');
 module.exports = React.createClass({
 
   propTypes: {
-    purpose: React.PropTypes.oneOf(['primary', 'secondary', 'success', 'warning', 'danger', 'info']),
-    size: React.PropTypes.oneOf(['small', 'large', 'extra-large']),
+    purpose: React.PropTypes.oneOf(['defult', 'primary', 'secondary', 'success', 'warning', 'danger', 'info']),
```

Comment:
Do we want "default" to be an option or should we instead use `getDefaultProps` to specify that "medium" is the default if none is provided? this would remove the need for the `.medium, .default` in the less below.